src/assets/NcAppNavigationItem.scss

```diff
@@ -132,11 +132,12 @@
 	flex-direction: column;
 	width: 100%;
 	gap: var(--default-grid-baseline, 4px);
+	padding-inline-start: 10px;
 
 	.app-navigation-entry {
 		display: inline-flex;
 		flex-wrap: wrap;
-		padding-inline-start: $icon-size;
+		padding-inline-start: 10px;
```

Comment:
i changed this to 10px, instead of 16px (icon-size), because the indentation is unnecessarily large.